lib/dw/visualization.base.js

```diff
@@ -317,9 +317,6 @@ extend(base, {
         this.clear();
         const el = document.querySelector('#chart');
         el.innerHTML = '';
-        // this removes all event handlers
-        // eslint-disable-next-line
-        el.outerHTML = el.outerHTML;
```

Comment:
I'm not sure why exactly this hack was needed, but it interferes with Svelte references, so I had to removed it.

Reply:
before refactoring this method, we have used this jQuery code to remove event handlers https://github.com/datawrapper/datawrapper/blob/4acb0ec4e931305e3de34a4add9457045c0bbe53/dw.js/src/dw.visualization.base.js#L315-L325

hopefully this is no longer necessary